### ISSUE TITLE
pretrained weights

### ISSUE THREAD

**Issue by xyeahr** [DetectionTeamUCAS/FPN_Tensorflow#7]
When I trained my model with pretrained-model, the variables are zeros like this :

model restore from pretrained mode, path is : /home/..//data/pretrained_weights/resnet_50.ckpt
resnet_v1_50/conv1/weights:0
resnet_v1_50/conv1/BatchNorm/gamma:0
resnet_v1_50/conv1/BatchNorm/beta:0
resnet_v1_50/conv1/BatchNorm/moving_mean:0
resnet_v1_50/conv1/BatchNorm/moving_variance:0
resnet_v1_50/block1/unit_1/bottleneck_v1/shortcut/weights:0
resnet_v1_50/block1/unit_1/bottleneck_v1/shortcut/BatchNorm/gamma:0
resnet_v1_50/block1/unit_1/bottleneck_v1/shortcut/BatchNorm/beta:0
resnet_v1_50/block1/unit_1/bottleneck_v1/shortcut/BatchNorm/moving_mean:0
resnet_v1_50/block1/unit_1/bottleneck_v1/shortcut/BatchNorm/moving_variance:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv1/weights:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv1/BatchNorm/gamma:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv1/BatchNorm/moving_mean:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv1/BatchNorm/moving_variance:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv2/weights:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv2/BatchNorm/gamma:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv2/BatchNorm/beta:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv2/BatchNorm/moving_mean:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv2/BatchNorm/moving_variance:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv3/weights:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv3/BatchNorm/gamma:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv3/BatchNorm/beta:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv3/BatchNorm/moving_mean:0
resnet_v1_50/block1/unit_1/bottleneck_v1/conv3/BatchNorm/moving_variance:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv1/weights:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv1/BatchNorm/gamma:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv1/BatchNorm/beta:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv1/BatchNorm/moving_mean:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv1/BatchNorm/moving_variance:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv2/weights:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv2/BatchNorm/gamma:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv2/BatchNorm/beta:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv2/BatchNorm/moving_mean:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv2/BatchNorm/moving_variance:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv3/weights:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv3/BatchNorm/gamma:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv3/BatchNorm/beta:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv3/BatchNorm/moving_mean:0
resnet_v1_50/block1/unit_2/bottleneck_v1/conv3/BatchNorm/moving_variance:0
resnet_v1_50/block1/unit_3/bottleneck_v1/conv1/weights:0
...
...
